cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/concepts/PythonLoggingConceptPass.kt

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) 2024, Fraunhofer AISEC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+package de.fraunhofer.aisec.cpg.passes.concepts
+
+import de.fraunhofer.aisec.cpg.TranslationContext
+import de.fraunhofer.aisec.cpg.graph.*
+import de.fraunhofer.aisec.cpg.graph.concepts.logging.LogLevel
+import de.fraunhofer.aisec.cpg.graph.concepts.logging.LoggingNode
+import de.fraunhofer.aisec.cpg.graph.concepts.logging.newLogOperationNode
+import de.fraunhofer.aisec.cpg.graph.concepts.logging.newLoggingNode
+import de.fraunhofer.aisec.cpg.graph.declarations.ImportDeclaration
+import de.fraunhofer.aisec.cpg.graph.statements.expressions.CallExpression
+import de.fraunhofer.aisec.cpg.graph.statements.expressions.MemberExpression
+import de.fraunhofer.aisec.cpg.passes.ComponentPass
+import de.fraunhofer.aisec.cpg.passes.configuration.ExecuteLate
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+/**
+ * This pass collects Python's `import logging` logs and maps them to the corresponding CPG
+ * [Concept] nodes.
+ */
+@ExecuteLate
```

Comment:
Rather than having execute later, wouldn't it be better to explicitly state the dependencies it has, e.g. -> SymbolResolver/EOG?

Reply:
> I don't think so. I believe the concept passes build upon "the cpg", thus `ExecuteLate` is exactly correct. We can add more dependencies like "depends on other concept pass" or depends on "symbolresolver/eog" to make this more explicit, but I'd like to keep `ExecuteLate`

This is not necessarily true. We do have some concept passes that are executed in-between "normal" CPG passes, for example concepts about dynamic loading. So not every concept pass should have execute late.